From aa78815b05090df63bef9ac1b6473f32aac5363d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Kundr=C3=A1t?= <jan.kundrat@cesnet.cz>
Date: Tue, 18 Mar 2025 11:25:05 +0100
Subject: [PATCH 11/20] Fix a typo
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Organization: Wires

...because I don't want to go through the CI again, hence a separate
commit.

Fixes: 6ed5442 Utilities for working with existing sysrepo:discard-items nodes
Reported-by: Tomáš Pecka <tomas.pecka@cesnet.cz>
Change-Id: I31f3efc1006fe222b7853b24a116b3d8cc04dc8d
Signed-off-by: Mattias Walström <lazzer@gmail.com>
---
 src/utils/utils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/utils.cpp b/src/utils/utils.cpp
index e339e88..ff4a32c 100644
--- a/src/utils/utils.cpp
+++ b/src/utils/utils.cpp
@@ -95,7 +95,7 @@ std::optional<libyang::DataNode> findMatchingDiscard(libyang::DataNode root, con
 }
 
 /**
- * @short Find all sysrepo:discard-items nodes which match the given XPath or the descendats of this XPath
+ * @short Find all sysrepo:discard-items nodes which match the given XPath or the descendants of this XPath
  */
 std::vector<libyang::DataNode> findMatchingDiscardPrefixes(libyang::DataNode root, const std::string& xpathPrefix)
 {
-- 
2.43.0

